trending_data/src/main/webapp/unique_window_handler.js

```diff
@@ -111,25 +112,23 @@ export class UniqueWindowHandler {
     }
   }
 
-  /**
-  * Checks if the current window is open
-  * @param {Marker} marker
-  * @return {boolean}
-  */
-  isClickOnDifferentWindow(marker) {
-    return (!this.isInfoWindowOpen() ||
-      marker.countryCode != this.getCountryCode());
-  }
-
   /**
   * Loads the current YouTube data and opens a window
   * which contains it
   * @param {Marker} marker
-  * @param {HTMLElement} content
   */
   async openWindow(marker) {
-    const YTcontent = await this.loadYTData(marker);
-    this.loadTwitterData(marker);
+    this.marker = marker;
+    this.currentCode = this.marker.countryCode;
+    if (this.currentCode == this.lastCode) {
+      this.lastCode = this.currentCode;
+      return;
+    }
```

Comment:
Bug : Can't open the window of the country immediately after closing it.

Reply:
Can be fixed by adding a method that gets called on close of the window that resets the code to null.

Reply:
Also we don't really need currentCode and lastCode. We can have just one of them and the other is in the marker.

Reply:
Fixed